jwst/ramp_fitting/ramp_fit_step.py

```diff
@@ -89,5 +89,7 @@ def process(self, input):
         if int_model is not None:
             int_model.meta.cal_step.ramp_fit = 'COMPLETE'
 
-        return out_model, int_model
+        if input_model.meta.exposure.type in ('NRS_IFU', 'MIR_MRS'):
+            out_model = datamodels.IFUImageModel(out_model)
```

Comment:
If out_model was originally created within ramp_fit as ImageModel, does this just magically convert it to an IFUImageModel? Cool (didn't know you could do that).

Reply:
Yes, it does. You can't do the opposite though.